includes/API/FBE/Configuration/Read/Response.php

```diff
@@ -43,7 +43,7 @@ public function get_messenger_configuration() {
 	/**
 	 * Is Instagram Shopping enabled?
 	 *
-	 * @since x.x.x
+	 * @since 2.6.0
```

Comment:
For some reason these docblock version number changes aren't on the `2.6.0` tag. Ignoring since this shouldn't cause any issues.